understanding/22/accessible-authentication.html

```diff
@@ -0,0 +1,120 @@
+        </blockquote>
+    </section>
+   
+    <section id="intent">
+        <h2>Intent of Accessible Authentication</h2>
+
+        <p>The purpose of this success criterion is to ensure there is an accessible, easy-to-use, secure method to log in and access content. Most websites rely on usernames and passwords for logging in. Memorizing a username and password (or transcribing it manually) places a very high or impossible burden upon people with certain cognitive disabilities. </p>
+ 
+        <p>Cognitive function tests are known to be problematic for many people with cognitive disabilities. Whether it is remembering random strings of characters, a pattern gesture to perform on a touch screen, or identifying which images include a particular object, this form of test will exclude some people. When a cognitive function test is used, that at least one other authentication method must available which uses does not use a cognitive function test.</p>
+            
+        <p class="note">Websites can employ username (or email) and password inputs as an authentication method if it meets <a href="#">Success Criterion 1.3.5 Input Purpose</a> and password managers / browser functionality is not blocked. When properly marked up, browser features or password managers can save the user’s information and refill the login.</p>
+
+        <p>If there is more than one step in the authentication process, such as with multi-factor authentication, all steps should comply with this success criterion. There should be a path through authentication that does not rely on cognitive function tests.</p>
+
+
+
+    </section>
+    <section id="benefits">
+      <h2>Benefits of Accessible Authentication</h2>
+      
+      <p>People with cognitive issues to do with memory, reading (e.g. dyslexia), numbers (e.g. dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.</p>
+      
+   </section>
+   
+   <section id="examples">
+      <h2>Examples of Accessible Authentication</h2>
+
+      <ul>
+         <li>A website uses a properly marked up email and password as the login authentication.</li>
+         <li>A website uses WebAuthn so the user can authenticate with their device instead of username/password. The user’s device could use any available modality. Common methods on laptops and phones are facial-scan, fingerprint, and pin number. The website is not enforcing any particular use, it is assumed a user will setup a method that suits them.</li>
+         <li>A website offers the ability to login with a 3rd party provider using the OAuth method.</li>
+         <li>A website that requires 2-factor authentication allows for multiple options for the 2nd factor, including a USB-based method where the user simply presses a button to enter a time-based token.</li>
```

Comment:
Prose seems to require USB 2FA via pushbutton. Was the intention that this should be an optional example? If so, it should be reworded.

Reply:
ditto

Reply:
I tried to think of some line of text to put at the top, but the heading starting "Examples" made it redundant.